packages/advanced-logic/src/extensions/payment-network/near-native.ts

```diff
@@ -27,6 +27,7 @@ export default class NearNativePaymentNetwork extends NativeTokenPaymentNetwork
       case 'aurora':
         return this.isValidMainnetAddress(address);
       case 'aurora-testnet':
+      case 'near-testnet':
```

Comment:
I think if you add this case we should increase the CURRENT_VERSION. `near-testnet` is not valid in `0.2.0`, both are valids in `0.3.0` and we can then invalidate `aurora` in `0.4.0`.

Reply:
that's why to avoid that I will add the temporary mapping as for any-to-native